jcl/src/java.base/share/classes/java/lang/Class.java

```diff
@@ -541,7 +547,9 @@ public static Class<?> forName(
 		}
 	}
 	Class<?> c = forNameImpl(className, false, classLoader);
+	/*[IF JAVA_SPEC_VERSION < 24]*/
```

Comment:
This isn't needed since it is within `/*[ELSE] JAVA_SPEC_VERSION >= 18`.

Reply:
Good catch, thanks.